android/src/app/organicmaps/MwmActivity.java

```diff
@@ -363,6 +369,32 @@ public int getThemeResourceId(@NonNull String theme)
     return super.getThemeResourceId(theme);
   }
 
+  private void replaceMapWithPlaceholder()
+  {
+    disableControls();
+    if (mMapFragment != null)
+    {
+      getSupportFragmentManager().beginTransaction().remove(mMapFragment).commitNowAllowingStateLoss();
+      mMapFragment = null;
+    }
+    getSupportFragmentManager().beginTransaction().replace(R.id.map_fragment_container, new MapPlaceholderFragment()).commitNowAllowingStateLoss();
```

Comment:
`commitNow` to force push changes. If I use `commit`, changes will be applied with a delay that breaks the destruction and creation of surfaces:
Call order with `commitNow`:
* Device: remove `mapFragment`
* Device: destroy surface (triggered by commitNow)
* Car: create a new surface

Call order with `commit`:
* Device: remove `mapFragment`
* Car: create a new surface
* Device: destroy surface (triggered by commit)

Or even "better":
* Car: create a new surface
* Device: remove `mapFragment`
* Device: destroy surface (triggered by commit)


`AllowingStateLoss` is used because it may be called when the app is hidden on the device:
* Map is shown on the device
* Switch to another app
* Car: use a map on the car

It's impossible to switch fragments after `onSaveInstanceState` that's why `AllowingStateLoss` is used